intro_flutter_gpu/step_09/lib/main.dart

```diff
@@ -198,4 +148,42 @@ class TrianglePainter extends CustomPainter {
 
   @override
   bool shouldRepaint(covariant CustomPainter oldDelegate) => true;
+
+  static const List<List<double>> vertices = [
+    // Vertex format: [x, y, z, u, v, w]
+
+    // Front face of cube
+    [-0.5, -0.5, 0.5, 0.0, 0.0, 1.0],
+    [0.5, -0.5, 0.5, 1.0, 0.0, 1.0],
+    [0.5, 0.5, 0.5, 1.0, 1.0, 1.0],
+    [-0.5, 0.5, 0.5, 0.0, 1.0, 1.0],
+
+    // back face of cube
```

Comment:
To be consistent with the front face comment:

```suggestion
    // Back face of cube
```